packages/devextreme/testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js

```diff
@@ -1565,19 +1564,16 @@ QUnit.module('common use cases', {
             useSelectMode: true,
             selectedItemKey: 1
         });
-
         const items = [{ id: 3, name: 'test' }];
         this.dropDownButton.option('items', items);
 
-        setTimeout(() => {
```

Comment:
We repaired the test => let's restore its initial state? Now we don't have a reason to change it.
If u insist in its change, let's make it for all tests added in that commit and ensure that it's not evergreen.